python/hsfs/storage_connector.py

```diff
@@ -192,6 +192,11 @@ def expiration(self):
         """Cluster temporary credential expiration time."""
         return self._expiration
 
+    @property
+    def bucket(self):
+        """Return the bucket for S3 connectors."""
+        return self._bucket
```

Comment:
should we add the same check as for "path" if it's actually a S3 connector? Since I guess for JDBC a bucket doesn't make sense

Reply:
Yes, we should probably raise errors if the storage connector is used incorrectly. I'll fix that

Reply:
Done